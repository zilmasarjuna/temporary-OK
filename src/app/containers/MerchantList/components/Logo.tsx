import React from 'react';
import styled from 'styled-components/macro';

import LogoOttopay from 'app/assets/icon-logo-ottopay.png';

const ImgLogo = styled.div`
  margin: 0 auto 36px;
  text-align: center;

  img {
    width: 64px;
  }
`;

const Logo = () => (
  <ImgLogo>
    <img src={LogoOttopay} alt="img-ottopay" />
  </ImgLogo>
);

export default Logo;
