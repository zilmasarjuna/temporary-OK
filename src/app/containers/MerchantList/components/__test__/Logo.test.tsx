import React from 'react';
import { shallow } from 'enzyme';

import Logo from '../Logo';

describe('<Logo /> in Card Merchant', () => {
  it('has a image', () => {
    const wrapper = shallow(<Logo />);
    const img = wrapper.find('img');
    expect(img).toHaveLength(1);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
