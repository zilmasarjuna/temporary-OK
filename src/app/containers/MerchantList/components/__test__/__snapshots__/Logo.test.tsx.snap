// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> in Card Merchant has a image 1`] = `
"<Logo__ImgLogo>
  <img src=\\"icon-logo-ottopay.png\\" alt=\\"img-ottopay\\" />
</Logo__ImgLogo>"
`;
